conf/application.conf

```diff
@@ -250,3 +252,22 @@ hmrc-timeout-dialog {
   enableSynchroniseTabs = true
   defaultCountdownInSeconds = 120
 }
+
+sca-wrapper {
+    future-timeout = 5
+    host = "http://localhost:9514" # used in feedback url
+    feedback-service-name = "CYSP/pension"
+    exit-survey-origin = "CYSP"
+    accessibility-statement {
+      referrer.url = "/check-your-state-pension"
+      redirect.url = "check-your-state-pension"
+    }
+    service-name {
+      messages-key = "Check or update your medical insurance benefit"
+    }
+    signout {
+      url = "/check-your-state-pension/sign-out"
+      alternative-url = null
+    }
```

Comment:
Is it not already in the template as an argument?

I have the same issue in benefits-frontend so I don't remember if it was intended or just a mistake.